jdi-light/src/main/java/com/epam/jdi/light/common/PropertyValidator.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.jdi.light.common;
+
+import java.security.InvalidParameterException;
+import java.util.Properties;
+
+public class PropertyValidator {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'PropertyValidator' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4532609)